mmv1/products/cloudrunv2/api.yaml

```diff
@@ -0,0 +1,715 @@
+                - !ruby/object:Api::Type::NestedObject
+                  name: "cloudSqlInstance"
+                  description: |-
+                    For Cloud SQL volumes, contains the specific instances that should be mounted. Visit https://cloud.google.com/sql/docs/mysql/connect-run for more information on how to connect Cloud SQL and Cloud Run.
+                  # exactly_one_of:
+                  #   - template.0.volumes.0.secret
+                  #   - template.0.volumes.0.cloudSqlInstance
+                  properties:
+                    - !ruby/object:Api::Type::Array
+                      name: "instances"
+                      description: |-
+                        The Cloud SQL instance connection names, as can be found in https://console.cloud.google.com/sql/instances. Visit https://cloud.google.com/sql/docs/mysql/connect-run for more information on how to connect Cloud SQL and Cloud Run. Format: {project}:{location}:{instance}
+                      item_type: Api::Type::String
+          - !ruby/object:Api::Type::Enum
+            name: "executionEnvironment"
+            description: |-
+              The sandbox environment to host this Revision.
+            values:
+              - :EXECUTION_ENVIRONMENT_GEN1
+              - :EXECUTION_ENVIRONMENT_GEN2
+          - !ruby/object:Api::Type::String
+            name: "encryptionKey"
+            description: |-
+              A reference to a customer managed encryption key (CMEK) to use to encrypt this container image. For more information, go to https://cloud.google.com/run/docs/securing/using-cmek
+          - !ruby/object:Api::Type::Integer
+            name: "maxInstanceRequestConcurrency"
+            description: |-
+              Sets the maximum number of requests that each serving instance can receive.      
+      - !ruby/object:Api::Type::Array
+        name: "traffic"
+        description: |-
+          Specifies how to distribute traffic over a collection of Revisions belonging to the Service. If traffic is empty or not provided, defaults to 100% traffic to the latest Ready Revision.
+        item_type: !ruby/object:Api::Type::NestedObject
+          properties:
+          - !ruby/object:Api::Type::Enum
+            name: "type"
+            description: |
+              The allocation type for this traffic target.
+            values:
+              - :TRAFFIC_TARGET_ALLOCATION_TYPE_LATEST
+              - :TRAFFIC_TARGET_ALLOCATION_TYPE_REVISION
+          - !ruby/object:Api::Type::String
+            name: "revision"
+            description: |
+              Revision to which to send this portion of traffic, if traffic allocation is by revision.
+          - !ruby/object:Api::Type::Integer
+            name: "percent"
+            description: |
+              Specifies percent of the traffic to this Revision. This defaults to zero if unspecified.
```

Comment:
possibly this should be `default_from_api: true`?

Reply:
Good catch! Added!

Reply:
Oh wait. Since the default is 0, it may not run into perma-diff though.